sphinx/source/docs/user_guide/tools.rst

```diff
@@ -689,40 +684,40 @@ that are to be edited must be supplied explicitly as a list:
     r2 = p.rect('x', 'y', 'width', 'height', source=source2)
     tool = BoxEditTool(renderers=[r1, r2])
 
-The tool will automatically modify the columns on the data source
+The tool automatically modifies the columns on the data source
 corresponding to the ``x``, ``y``, ``width``, and ``height`` values of
 the glyph. Any additional columns in the data source will be padded
-with the declared ``empty_value``, when adding a new box. When drawing
+with the declared ``empty_value`` when adding a new box. When drawing
 a new box, the data will always be added to the ``ColumnDataSource`` on
 the first supplied renderer.
 
-It is also often useful to limit the number of elements that can be
+It is often useful to limit the number of elements that can be
 drawn, e.g. when specifying a specific number of regions of interest.
```

Comment:
```suggestion
drawn. For example, when you want to specify certain regions of interest.
```

Reply:
Well, thinking about it, I wonder if we should be a little clearer with what we mean by this. How about:

> For example, when you want to limit the option to use the BoxEditTool to certain regions of interest.

Did I understand the idea here correctly, @bryevdv?